apps/storybook-react/stories/Menu.stories.tsx

```diff
@@ -191,97 +180,16 @@ const bigMenuTemplate = (
   </>
 )
 
-export const EdgeDetection = () => {
-  const [state, setState] = React.useState({
-    topLeft: null,
-    topRight: null,
-    bottomLeft: null,
-    bottomRight: null,
-  })
-
-  const topLeftRef = React.useRef()
-  const topRightRef = React.useRef()
-  const bottomLeftRef = React.useRef()
-  const bottomRightRef = React.useRef()
-
-  useEffect(() => {
-    setState({
-      topLeft: topLeftRef.current,
-      topRight: topRightRef.current,
-      bottomLeft: bottomLeftRef.current,
-      bottomRight: bottomRightRef.current,
-    })
-  }, [state.topLeft])
-
-  const { topLeft, topRight, bottomLeft, bottomRight } = state
-
-  return (
-    <Grid>
-      <FloatingAnchor
-        id="anchor-topleft"
-        aria-controls="menu-topleft"
-        aria-haspopup="true"
-        ref={topLeftRef}
-        style={{ left: 0, top: 0 }}
-      >
-        Top left
-      </FloatingAnchor>
-      <Menu id="menu-topleft" open={Boolean(topLeft)} anchorEl={topLeft}>
-        {simpleMenuTemplate}
-      </Menu>
-
-      <FloatingAnchor
-        id="anchor-topright"
-        aria-controls="menu-topright"
-        aria-haspopup="true"
-        ref={topRightRef}
-        style={{ top: 0, right: 0 }}
-      >
-        Top Right
-      </FloatingAnchor>
-      <Menu id="menu-topright" open={Boolean(topRight)} anchorEl={topRight}>
-        {simpleMenuTemplate}
-      </Menu>
-
-      <FloatingAnchor
-        id="anchor-bottomleft"
-        aria-controls="menu-bottomleft"
-        aria-haspopup="true"
-        ref={bottomLeftRef}
-        style={{ bottom: 0, left: 0 }}
-      >
-        Bottom Left
-      </FloatingAnchor>
-      <Menu
-        id="menu-bottomleft"
-        open={Boolean(bottomLeft)}
-        anchorEl={bottomLeft}
-      >
-        {simpleMenuTemplate}
-      </Menu>
-
-      <FloatingAnchor
-        id="anchor-bottomright"
-        aria-controls="menu-bottomright"
-        aria-haspopup="true"
-        ref={bottomRightRef}
-        style={{ bottom: 0, right: 0 }}
-      >
-        Bottom right
-      </FloatingAnchor>
-      <Menu
-        id="menu-bottomright"
-        open={Boolean(bottomRight)}
-        anchorEl={bottomRight}
-      >
-        {simpleMenuTemplate}
-      </Menu>
-    </Grid>
-  )
-}
+export default {
+  title: 'Components/Menu',
+  component: Menu,
```

Comment:
Try using something like this to fix menus in story?
```suggestion
  component: Menu,
  decorators: [
    (Story) => (
      <div style={{ margin: '3rem' }}>
        <Story />
      </div>
    ),
  ],
```